layouts/partials/comment.html

```diff
@@ -211,8 +211,13 @@
             {{- $options := dict -}}
             {{- $options = dict "targetPath" "js/twikoo.min.js" | merge $options -}}
             {{- $options = dict "minify" true | merge $options -}}
-            {{- $source := $cdn.twikooJS | default "lib/twikoo/twikoo.all.min.js" -}}
-            {{- dict "Source" $source "Fingerprint" $fingerprint "Defer" true | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+            {{- if eq $twikoo.cloudbase true -}}
+                {{- $source := $cdn.twikooAllJS | default "lib/twikoo/twikoo.all.min.js" -}}
+                {{- dict "Source" $source "Fingerprint" $fingerprint "Defer" true | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+                {{- else -}}
+                {{- $source := $cdn.twikooJS | default "lib/twikoo/twikoo.min.js" -}}
+                {{- dict "Source" $source "Fingerprint" $fingerprint "Defer" true | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+            {{- end -}}

```

Comment:
I am slightly worried about backwards compatibility here. People who are currently using cloudbase will have to add the config on upgrading the theme. Is it possible that we make the default package `twikoo.all.min.js` and people can opt in to use the smaller one if they are not using cloudbase.

Reply:
Yes, we can set cloudbase to true in the config.

Reply:
updated